.github/actions/build/action.yml

```diff
@@ -0,0 +1,141 @@
+        Start-Process -NoNewWindow -Wait -FilePath "curl.exe" -ArgumentList "-L -o `"$swigZip`" `"$swigUrl`""
+        if (!(Test-Path $swigZip) -or ((Get-Item $swigZip).Length -lt 500KB)) { Write-Host "Download failed or file is corrupted." }
+        Expand-Archive -Path $swigZip -DestinationPath $swigDir -Force
+
+    - name: "Add Basilisk and SWIG paths"
+      if: runner.os == 'Windows'
+      shell: pwsh
+      run: |
+        $oldpath = (Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).path
+        $newPath = “C:\Program Files\SWIG\swigwin-4.2.1;$oldpath;${{ env.GITHUB_WORKSPACE }}\dist3\Basilisk”
+        echo "PATH=$newPath" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+
+    - name: Update Python tooling to latest
+      shell: bash
+      run: pip install -U pip wheel setuptools
+
+    - name: Install requirements
+      shell: bash
+      run: |
+        if [[ "${{ inputs.is-canary }}" == 'true' ]]; then
+          pip install -r .github/workflows/requirements.txt -r .github/workflows/requirements_dev.txt
+        else
+          pip install -r requirements.txt -r requirements_dev.txt
+        fi
+
+    - name: Install documentation requirements
+      if: ${{ inputs.install-doc-reqs == 'true' }}
+      shell: bash
+      run: |
+        if [[ "${{ inputs.is-canary }}" == 'true' ]]; then
+          pip install -r .github/workflows/requirements_doc.txt
+        else
+          pip install -r requirements_doc.txt
+        fi
+
+    - name: Cache Conan
+      if: ${{ inputs.build-mode == 'conan' }}
+      uses: actions/cache@v4
+      with:
+        path: ${{ env.HOME }}/.conan2
+        key: >-
+          conan-${{ runner.os }}-py${{ inputs.python-version }}-
+          ${{ hashFiles('conan.lock','conanfile.*','**/conanfile.*','CMakeLists.txt','**/*.cmake') }}
+        restore-keys: |
+          conan-${{ runner.os }}-py${{ inputs.python-version }}-
+
+    - name: Configure Conan profile
+      if: ${{ inputs.build-mode == 'conan' }}
+      shell: bash
+      run: |
+        python -m conans.conan profile detect --exist-ok
+        prof_path="$(python -m conans.conan profile path default)"
+
+        grep -q '^\[conf\]' "$prof_path" || printf "\n[conf]\n" >> "$prof_path"
+        grep -q '^tools.system.package_manager:mode=install$' "$prof_path" || \
+          printf "tools.system.package_manager:mode=install\n" >> "$prof_path"
+        grep -q '^tools.system.package_manager:sudo=True$' "$prof_path" || \
+          printf "tools.system.package_manager:sudo=True\n" >> "$prof_path"
+
+    - name: Build Basilisk (Conanfile)
+      if: ${{ inputs.build-mode == 'conan' }}
+      shell: bash
+      env:
+        CONAN_NON_INTERACTIVE: "1"
+      run: |
+        python conanfile.py ${{ inputs.conan-args }}
+
+    - name: Build Basilisk (Pip)
```

Comment:
You mentioned the conan way of building is still being implicitly tested with the pip build process, correct?